content/about/faq.md

```diff
@@ -107,7 +107,7 @@ into the Haiku code base. The rest is either homebuilt code or derivatives of ex
 is mostly compatible with the BeOS API. In other words, many applications that were initially developed for BeOS work flawlessly on Haiku.
 
 ### I've never seen Haiku. What does it look like?
-Check out our [Haiku Slideshow](/slideshows/) and the [Haiku Movies](/about/movies) pages.
+Check out our [Haiku Slideshow](/slideshows/haiku-1/) and the [Haiku Movies](/about/movies) pages.
```

Comment:
The [about page](https://github.com/haiku/website/blob/master/content/about/_index.md) links to /slideshows/haiku-tour which rewrites to this particular URL. I do not know which one is preferred.

Reply:
haiku-tour makes more sense. haiku-1 is just the first page of the "tour".

Reply:
Oh ok. I'll fix that.

Reply:
Done.